rooms.js

```diff
@@ -261,6 +261,22 @@ class Room {
 			}
 		}
 	}
+
+	get metadata() {
+		let settings = {
+			// settings
+			privacy: (this.isPrivate === true ? 'secret' : this.isPrivate === 'hidden' ? 'hidden' : 'public'),
+			modchat: this.modchat,
+			modjoin: this.modjoin,
+
+			// games
```

Comment:
Yeah, but, in hindsight, it's a bit much to send to a million users a day when most of them aren't going to use it at all.